packages/validation/src/cloudpulse.schema.ts

```diff
@@ -1,44 +1,44 @@
 import { array, number, object, string } from 'yup';
 
+const fieldErrorMessage = 'This field is required.';
```

Comment:
full stop should not be there in error messages. Please change at all places
```suggestion
const fieldErrorMessage = 'This field is required';
```

Reply:
But full stop is there in error messages of other forms.

Reply:
@nikhagra-akamai , I remember there was a PR comment, stating we need fullstop for error messages

Reply:
okay, In our other components we haven't used full stop so I thought